autouri/cli.py

```diff
@@ -161,22 +185,33 @@ def main():
                 raise NotImplementedError
             logger.info('Copying from file {s} to {t} {reason}'.format(
                 s=src, t=target, reason=reason))
-
     elif args.action == 'read':
         s = AutoURI(src).read()
         print(s)
 
+    elif args.action == 'find':
+        for uri in AutoURI(src).find_all_files():
+            print(uri)
+
     elif args.action == 'write':
         AutoURI(src).write(args.text)
         logger.info('Text has been written to {s}'.format(s=src))
 
-    elif args.action == 'rm':        
+    elif args.action == 'rm':
         u = AutoURI(src)
         if not u.exists:
             raise ValueError('File does not exist. {s}'.format(s=src))
         u.rm()
         logger.info('Deleted {s}'.format(s=src))
 
+    elif args.action == 'rmdir':
+        AutoURI(src).rmdir(dry_run=not args.delete, num_threads=args.nth)
```

Comment:
With my proposed change to the parser this would be more legible:
```suggestion
        AutoURI(src).rmdir(dry_run=not args.delete, num_threads=args.num_threads)
```

Reply:
Fixed as you suggested.